docs/examples/index.md

```diff
@@ -1,8 +1,134 @@
 ---
-title: Basic usage
-sidebar_label: Basic usage
+sidebar_label: Examples
+title: 'Code examples'
 ---
 
-This section contains examples of how to use Apify Client for Javascript.
+## Different ways how to pass an input
 
-There is nothing here yet.
+The fastest way to get results from an Actor is to pass input directly to the `call` function.
```

Comment:
Can you maybe a bit describe what the example does? E.g. it starts an Actor that scrapers 20 posts from the Instagram website based on the hashtag ... (or something like that)

Reply:
@HonzaTuron Seems like this is still not resolved, please resolve the comments only AFTER you do the change and ideally reply to the thread, otherwise it's really hard to understand the thinking behind marking it as solved without any reaction.